tests/source/enum.rs

```diff
@@ -92,3 +92,7 @@ fn nested_enum_test() {
         }
     }
 }
+
+   pub  struct  EmtpyWithComment {
+    // TODO: Implement this struct
```

Comment:
Probably shouldn't use "TODO" here, it will trigger the TODO lint and we might watch for that some day.